doc/user/content/sql/update.md

```diff
@@ -10,14 +10,18 @@ menu:
 
 ## Syntax
 
-{{< diagram "update-stmt.svg" >}}
+```mzsql
+UPDATE <table_name> [ AS <alias> ]
+   SET <column1_name> = <expression1> [, <column2_name> = <expression2>, ...]
+[WHERE <condition> ];
+```
 
-Field | Use
+### Options
```

Comment:
Does this need a sub-section? If so, can you also add it to the other pages?

Reply:
Ah ... no need for the section.